account_financial_report_webkit/models/account_static_balance.py

```diff
@@ -286,7 +286,7 @@ def map_data_to_account_id(data):
                 SELECT * FROM account_journal_period ajp
                 WHERE ajp.period_id = asb.period_id
                     AND ajp.journal_id = asb.journal_id
-                    AND asb.create_date > ajp.write_date
+                    AND ajp.write_date < asb.create_date
```

Comment:
This part was not wrong, but I'm sorting out the `ajp` arguments to the left across the whole WHERE clause.